skimage/restoration/_denoise_cy.pyx

```diff
@@ -136,9 +136,9 @@ def _denoise_tv_bregman(np_floats[:, :, ::1] image, np_floats weight,
 
         rmse = 0
 
-        for r in range(1, rows + 1):
-            for c in range(1, cols + 1):
-                for k in range(dims):
+        for k in range(dims):
```

Comment:
:+1:

Reply:
:confused:  I meant :-1:
If the array is C contiguous in memory, previous loop order is the best!